packages/babel-core/README.md

```diff
@@ -108,8 +108,8 @@ Following is a table of the options you can use:
 | `moduleRoot`             | `(sourceRoot)`       | Optional prefix for the AMD module formatter that will be prepend to the filename on module definitions. |
 | `only`                   | `null`               | A [glob](https://github.com/isaacs/minimatch), regex, or mixed array of both, matching paths to **only** compile. Can also be an array of arrays containing paths to explicitly match. When attempting to compile a non-matching file it's returned verbatim. |
 | `parserOpts`             | `{}`                 | An object containing the options to be passed down to the babel parser, babylon |
-| `plugins`                | `[]`                 | List of [plugins](/docs/plugins/) to load and use. |
-| `presets`                | `[]`                 | List of [presets](/docs/plugins/#presets) (a set of plugins) to load and use. |
+| `plugins`                | `[]`                 | List of [plugins](https://babeljs.io/docs/plugins/) to load and use. |
+| `presets`                | `[]`                 | List of [presets](https://babeljs.io/docs/plugins/preset-latest/) (a set of plugins) to load and use. |
```

Comment:
@sethbergman I think this one is incorrect. Could you fix this to: http://babeljs.io/docs/plugins/#presets? Thanks 😬